src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -97,7 +84,7 @@ public bool ContentSet
         {
```

Comment:
Thinking this should most likely no longer be called `ContentSet`, `ContentModified` makes more sense.